src/components/carousel/index.js

```diff
@@ -164,6 +172,11 @@ export default class Carousel extends BaseComponent {
     return !loop && pageWidth;
   }
 
+  shouldEnablePagination() {
```

Comment:
I don't think the user needs to control if the paging is enabled or not. 
in a full-width page carousel it should always come with paging. 
I think it's a bug, not a feature..

Reply:
Why? In ScrollView the user controls it, why not in the Carousel? We can default it to true and the user can disable it...

Reply:
Cause the whole point of Carousel is the paging. 
Anyway, I'm fine with setting the default value to true.

Reply:
Done.